src/main/java/uk/gov/hmcts/darts/audio/service/impl/AdminMediaServiceImpl.java

```diff
@@ -263,23 +263,29 @@ public MediaApproveMarkedForDeletionResponse adminApproveMediaMarkedForDeletion(
         }
 
         mediaApproveMarkForDeletionValidator.validate(mediaId);
-        List<ObjectAdminActionEntity> objectAdminActionEntityList = objectAdminActionRepository.findByMedia_Id(mediaId);
-
         Optional<MediaEntity> mediaEntityOptional = mediaRepository.findByIdIncludeDeleted(mediaId);
         if (mediaEntityOptional.isEmpty()) {
             throw new DartsApiException(AudioApiError.MEDIA_NOT_FOUND);
         }
         MediaEntity mediaEntity = mediaEntityOptional.get();
-        var currentUser = userIdentity.getUserAccount();
-        var objectAdminActionEntity = objectAdminActionEntityList.getFirst();
-        objectAdminActionEntity.setMarkedForManualDeletion(true);
-        objectAdminActionEntity.setMarkedForManualDelBy(currentUser);
-        objectAdminActionEntity.setMarkedForManualDelDateTime(currentTimeHelper.currentOffsetDateTime());
-        objectAdminActionRepository.save(objectAdminActionEntity);
 
-        auditApi.record(AuditActivity.MANUAL_DELETION, currentUser, objectAdminActionEntity.getId().toString());
+        List<MediaEntity> mediaEntities = mediaRepository.findAllByChronicleId(mediaEntity.getChronicleId());
+
+        var currentUser = userIdentity.getUserAccount();
+        List<ObjectAdminActionEntity> objectAdminActionEntities = mediaEntities.stream()
+            .map(media -> media.getObjectAdminAction())
+            .filter(Optional::isPresent)
+            .map(Optional::get)
+            .peek(objectAdminAction -> {
+                objectAdminAction.setMarkedForManualDeletion(true);
+                objectAdminAction.setMarkedForManualDelBy(currentUser);
+                objectAdminAction.setMarkedForManualDelDateTime(currentTimeHelper.currentOffsetDateTime());
+                auditApi.record(AuditActivity.MANUAL_DELETION, currentUser, objectAdminAction.getId().toString());
+            })
```

Comment:
As per my comment on the integration test, this needs to ensure that the medias exist in the object_admin_action table as well.